packages/e2e.test.foundry-sdk-generator/generateMockOntology.js

```diff
@@ -107,10 +117,54 @@ export async function setup() {
     _: [],
     $0: "",
   });
+
+  await safeStat(testApp2Dir, "should exist");
 }
 
 export async function teardown() {
   // eslint-disable-next-line no-console
-  console.log("Test teardown: stopping API server");
+  console.log("teardown: stopping API server");
   apiServer.close();
 }
+
+await setup();
+await teardown();
+
+/**
+ * @param {string} testAppDir
+ */
+async function rmRf(testAppDir) {
+  try {
+    await fs.rm(testAppDir, { recursive: true });
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.debug("rm error", e);
+    // Only needed for regenerations
+  }
+}
+
+/**
+ * @param {string} filePath
+ * @param {"should exist" | "should not exist"} type
+ * @returns
+ */
+async function safeStat(filePath, type) {
```

Comment:
`fs.stat` throws if the file does not exist, which i generally don't care for as it makes ones code far more difficult to read, especially when you expect to be able to test for the non-existence of something. This helper both logs out whats going on as well as acting as an assert to clean up the rest of the codebase